pkg/proxy/proxy.go

```diff
@@ -72,7 +77,16 @@ func (p *SprayProxy) HandleProxy(c *gin.Context) {
 			continue
 		}
 		newRequest.Header = copy.Request.Header
+		// currently not distinguishing between requests we send and requests that return without error
+		metrics.IncForwardedCounter()
```

Comment:
No label at this point due to cardinality concerns. A single histogram already creates a large number of time series:

1. One per bucket (4 per my recommendation)
2. The sum of all observations
3. The count of all observations